scripts/utils/trading_strategy_helpers.js

```diff
@@ -550,12 +607,16 @@ const buildTransferFundsToMaster = async function(masterAddress, withdrawals, li
  * @return {Transaction} Multisend transaction that has to be sent from the master address to transfer back the funds stored in the exchange
  */
 const buildWithdrawAndTransferFundsToMaster = async function(masterAddress, withdrawals, web3 = web3, artifacts = artifacts) {
-  const withdrawalTransaction = await buildWithdraw(masterAddress, withdrawals, web3, artifacts)
-  const transferFundsToMasterTransaction = await buildTransferFundsToMaster(masterAddress, withdrawals, false, web3, artifacts)
+  const [withdrawalTransaction, transferFundsToMasterTransaction] = await Promise.all([
```

Comment:
Sometimes, I think that the original statement is much easier to read and likely the wins in waiting time might be minimal

Reply:
I tested it and I didn't see any meaningful impact in the running time (the difference, if it exists, is less than a second). I'll restore this function back to what it was.